### DISCUSSION TITLE
What is the source of entropy when `snap_getBip32Entropy` or `snap_getBip44Entropy` is called?

### DISCUSSION THREAD

**Discussion by ashutosh1206** [MetaMask/snaps#1773] [Q&A] [answered]
Is it only user-specific? Or is it user **and** snap specific?

If it's only user-specific, does that mean two snaps with same BIP-32 derivation path (eg. `["m", "44'", "60'"]`) and curve (eg. `secp256k1`) will be able to derive the same set of keys?

The documentation is not clear (https://docs.metamask.io/snaps/reference/rpc-api/#snap_getbip32entropy), and there's an open issue regarding this: https://github.com/MetaMask/snaps/issues/1611

**Comment by Montoya** [comment, marked as answer]
It is only user-specific. 

> If it's only user-specific, does that mean two snaps with same BIP-32 derivation path (eg. ["m", "44'", "60'"]) and curve (eg. secp256k1) will be able to derive the same set of keys?

That's correct.

**Reply by ashutosh1206** [reply]
Okay, thanks for the quick response!
